### PR TITLE
Form refacto

### PR DESCRIPTION
**Pull request by AlexandreBroudin**

### CODE REVIEW

File: src/scss/components/dropdown/dropdown.md [hunk 1]
```diff
@@ -1,29 +1,9 @@
...
-## Positions
-
-### Bottom-centered
-
-```html
-<div class="has-dropdown">
-  <button class="Btn" aria-controls="dropdownBottomCentered" aria-expanded="true">Call to action</button>
-  <ul id="dropdownBottomCentered" class="Dropdown Dropdown--bc">
+<div class="Dropdown" data-tapestry="dropdown">
+  <button class="Dropdown-toggle Btn" aria-controls="dropdownBottomLeft" aria-expanded="true">Call to action</button>
```
Comment (GabrielCousin) [inline src/scss/components/dropdown/dropdown.md]: `aria-expended = "false"` (in each case)

Comment (GabrielCousin) [inline src/scss/components/dropdown/dropdown.md]: Could we use something else to replace `dropdownBottomLeft`? Seems confusing to me.

File: src/scss/components/form/form.md [hunk 2]
```diff
@@ -1,34 +1,99 @@
...
-<label class="Label Label--inline" for="radioInput">Input Radio</label>
-```
+<fieldset class="Fieldset">
+  <legend class="Legend">Shipping Information</legend>
+  <label class="Label">Title</label>
+  <div class="FormRow Row">
+    <div class="Column Column-12">
+      <input name="title" id="mr" type="radio" class="Radio Input--inline" id="radioInput">
+      <label for="mr" class="Label Label--inline" for="radioInput">Mr</label>
+      <input name="title" id="mrs" for="" type="radio" class="Radio Input--inline" id="radioInput">
```
Comment (GabrielCousin) [inline src/scss/components/form/form.md]: Remove `for=""` ?

File: src/scss/components/form/form.md [hunk 3]
```diff
@@ -1,34 +1,99 @@
...
+          <li class="Dropdown-item" data-value="baz" data-label="Custom label 3">
+            Simple label 3
+          </li>
+        </ul>
+      </div>
+    </div>
+    <div class="Column Column-6">
+      <label class="Label">Country</label>
+      <div class="Dropdown Dropdown--block" data-tapestry="dropdown-select">
+        <input name="" value="bar" hidden>
```
Comment (GabrielCousin) [inline src/scss/components/form/form.md:61]: `type="hidden"` ?

**Comment by ugogo** [comment]
- Add a `padding-bottom: $pd--xs` on `.Legend`
- Not fan of `FormRow Row`
- Arrow icon of Dropdown-select

**Comment by AlexandreBroudin** [comment]
> Not fan of FormRow Row 

What do you suggest instead ?

> Arrow icon of Dropdown-select

Will take a look later, we need a better way to use dynamic icons

**Pull request merged**